Source/WebCore/rendering/style/StyleGradientImage.cpp

```diff
@@ -1397,21 +1547,15 @@ Ref<Gradient> StyleGradientImage::createGradient(const PrefixedRadialData& radia
 
 // MARK: - Deprecated Radial create.
 
-Ref<Gradient> StyleGradientImage::createGradient(const DeprecatedRadialData& radial, const RenderElement& renderer, const FloatSize& size, const RenderStyle& style) const
+Ref<Gradient> StyleGradientImage::createGradient(const DeprecatedRadialData& radial, const RenderElement&, const FloatSize& size, const RenderStyle& style) const
```

Comment:
Can we eliminate the `renderer` argument entirely instead of passing it and ignoring it?